Makefile

```diff
@@ -164,8 +166,8 @@ bundle: manifests kustomize
 	operator-sdk generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
 	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS) $(BUNDLE_FLAGS)
-	cp config/metadata/* bundle/metadata/
-	operator-sdk bundle validate ./bundle
+	cp config/metadata/* $(OUTPUT_DIR)/metadata/
```

Comment:
Oh I see! This directory was getting copied because the dependencies.yaml is located there. Good catch :)